extension/types/unicode-constants.ts

```diff
@@ -0,0 +1,17 @@
+export enum KANA {
```

Comment:
you could probably combine this with the lookup map if you called it something like kana_data.ts or kana_info.ts or something like that. Or since it's not all kana just character_info.ts.

Naming things is hard but there's a balance to strike on how fine grained groupings should be.

Reply:
I like that. `character_info.ts`